Created About Us Page

SuvayuNandy opened the pull request:
Created the about us page as a contribution under JWOC'25 as mentioned in issue #4 
![Screenshot from 2025-02-11 15-46-24](https://github.com/user-attachments/assets/e6916aca-5fb5-490d-be86-53c92661d4c7)

Sarita-021 commented:
@SuvayuNandy Can you please improve the ui a bit. Like colour of text, indentation and font style.

SuvayuNandy commented:
![image](https://github.com/user-attachments/assets/95474312-43a2-4d94-b36c-1d1b93550424)
Added some animations and also enhanced the colors and changed the font family as per your requirement

SuvayuNandy commented:
Hey @Sarita-021 you forgot to give the medium tag kindly give it

Sarita-021 commented:
@SuvayuNandy The UI you created is not attractive, and you have added only the text to it. That's why I changed the tag to easy.
